### ISSUE TITLE
[1630] Persist Pocket recommendations (instead of error/loading states)

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2186]
UX requirements from https://github.com/mozilla-mobile/firefox-tv/issues/2123#issuecomment-488500101. This issue must ship with Pocket channels.

### Why/User Benefit/User Problem
Sometimes users have flaky or slow connections: when they see loading or error states, it's a bad user experience. Instead, we can:
- Avoid a loading screen by updating the Pocket recommendations in the background and display the latest fetch (we currently do this except for first run)
- Avoid an error screen by persisting the latest Pocket recs to disk and displaying those; on first run, we display a set of recs bundled with the app

### What / Requirements
- [ ] (waiting on Pocket; expected end of week) Ship a bundled set of Pocket video recommendations for first run
  - #2154 tracks getting the content
- [ ] Fetch recommendations in the background (to minimize change, keep the current update schedule except defer the first update until 45min after the app is opened)
  - [ ] Ensure any changes to update scheme will not hammer Pocket servers
- [ ] Display the latest fetched recommendations to the user in the overlay (if we haven't fetched yet ^, display bundled Pocket video recs)
  - [ ] Never change displayed Pocket video tiles when the user has the overlay open (note: we currently implement this)
- [ ] Persist latest fetched recommendations ^ to disk so we have fresh recs to show on fresh start-up, rather than the bundled tiles

_If you need clarifications, please ask mcomella._

### Acceptance Criteria (how do I know when I’m done?)
- Pocket channel seems bundled like pinned tiles: no loading or error screens (except images, which can load dynamically)
- Pocket is usable to watch videos, works as a user might expect (e.g. back button works as expected)
- Pocket updates with new recommendations periodically

**Comment by athomasmoz** [comment]
Closing in favour of tracking under #2223
